ci-operator/step-registry/gcp/provision/minimal-permission/gcp-provision-minimal-permission-commands.sh

```diff
@@ -26,7 +30,39 @@ fi
 # The IAM service account for IPI: ipi-min-permissions-sa@${GOOGLE_PROJECT_ID}.iam.gserviceaccount.com
 # The IAM service account for UPI: upi-min-permissions-sa@${GOOGLE_PROJECT_ID}.iam.gserviceaccount.com
 # Currently we only deal with IPI in Prow CI.
-iam_account="ipi-min-permissions-sa@${GOOGLE_PROJECT_ID}.iam.gserviceaccount.com"
 
-gcloud iam service-accounts keys create "${SHARED_DIR}/gcp_min_permissions.json" --iam-account="${iam_account}" || exit 1
-echo "$(date -u --rfc-3339=seconds) - Created a temporary key of the IAM service account for the minimum permissions testing on GCP."
+sa_filename="ipi-min-permissions-sa.json"
+if [[ "${MINIMAL_PERMISSIONS_WITHOUT_ACT_AS}" == "yes" ]]; then
+    sa_filename="ipi-min-perm-without-actAs-sa.json"
+fi
+if [[ -f "${CLUSTER_PROFILE_DIR}/${sa_filename}" ]]; then
+  echo "$(date -u --rfc-3339=seconds) - Use pre-configured key of the IAM service account for the minimum permissions testing on GCP."
+  cp "${CLUSTER_PROFILE_DIR}/${sa_filename}" "${SHARED_DIR}/gcp_min_permissions.json"
+else
+  echo "$(date -u --rfc-3339=seconds) - ERROR Failed to find the pre-configured key file of the IAM service account '${iam_account}' in GCP project '${GOOGLE_PROJECT_ID}', abort." && exit 1
```

Comment:
The message referenced `iam_account`, should correct it a bit.

Reply:
Thank you @jianlinliu ! Not sure why the required tests did not catch it.